src/styles/index.scss

```diff
@@ -1,3 +1,118 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  color: $main-color;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  font-style: normal;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+
+  &__image {
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    font-size: 10;
```

Comment:
It seems like the font-size is missing a unit. According to the checklist, font-size values should include units like 'px', 'em', or 'rem'.